packages/policy/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+    "extends": "./node_modules/gts"
+  }
```

Comment:
```suggestion
 }
```

Reply:
Oh weird, this came straight from the `generate-bot` script.  I will take a look after we land this for sure.

Reply:
Addressed in #1516 as it affected my new bot too.